maas-schemas/schemas/tsp/booking-create/request.json

```diff
@@ -13,7 +13,23 @@
       "$ref": "http://maasglobal.com/core/booking.json#/definitions/terms"
     },
     "customer": {
-      "$ref": "http://maasglobal.com/core/customer.json"
+      "allOf": [
+        {
+          "$ref": "http://maasglobal.com/core/customer.json"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "documents": {
```

Comment:
I think this is not working, because `removeAdditional` removes documents on the previous pass and then this receives customer without `documents`.